.nvmrc

```diff
@@ -0,0 +1 @@
+v20.11.0
```

Comment:
I wanted to switch to a more modern node for CI / dev, but didn't want to force it in our package.json, so put it in `.nvmrc`. Open to feedback on this one tho cc/ @auscompgeek

Reply:
Isn't CI already testing on node 20?